02-intermediate-ruby/enumerable.md

```diff
@@ -126,3 +141,6 @@ In a group of 4, you shall research each of these `Enumerable` methods. Prepare
 - `group_by`
 - `zip`
 - `drop_while` & `take_while`
+
+## Additional Resources
+- For another interesting Mixin, look up [Comparable](https://ruby-doc.org/core-2.4.0/Comparable.html)
```

Comment:
Dan had a recommendation for me recently to remove the version number and just put "/core/" which will automatically point to the latest version